components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts.ts

```diff
@@ -1,12 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+
 import {MessageDescriptor} from 'react-intl';
 
-import {t} from '../../utils/i18n';
+import {t} from 'utils/i18n';
 
-export type KeyboardShortcutsSeq = MessageDescriptor | {default: MessageDescriptor; mac?: MessageDescriptor};
+export type NormalizeKeyboardShortcutDescriptor = MessageDescriptor | {default: MessageDescriptor; mac?: MessageDescriptor};
```

Comment:
this should be `NormalizedKeyboardShortcutDescriptor`, although I think you might have misunderstood my last review comment about renaming the normalize function.

IMO this could be named `KeyboardShortcutsDescriptor` as well